.github/workflows/validate-components.yml

```diff
@@ -31,4 +31,6 @@ jobs:
           export PATH="$PATH:$GOPATH/bin"
           cue vet -c ./schemas/windows_settings.cue ./vhdbuilder/packer/windows/windows_settings.json
           cue eval ./schemas/windows_settings.cue ./vhdbuilder/packer/windows/windows_settings.json
+          cue vet -c ./schemas/components.cue ./vhdbuilder/packer/windows/components-test.json
+          cue eval ./schemas/components.cue ./vhdbuilder/packer/windows/components-test.json
```

Comment:
We don't need the eval, just the vet I think. Not sure tho.

Reply:
No - the override is quite a different thing. What's where we get the override from aks-rp.